GherkinSyncTool.Synchronizers.TestRail/Client/TestRailClientWrapper.cs

```diff
@@ -47,14 +46,14 @@ public Case AddCase(CaseRequest caseRequest)
         public void UpdateCase(Case currentCase, CaseRequest caseToUpdate)
         {
             var policy = CreateResultHandlerPolicy<Case>();
-            var caseId = currentCase.Id ?? 
+            var caseId = currentCase.Id ??
                          throw new ArgumentException("Case Id cannot be null");
             if (!IsTestCaseContentEqual(caseToUpdate, currentCase))
             {
-                var updateCaseResult = policy.Execute(()=>
-                    _testRailClient.UpdateCase(caseId, caseToUpdate.Title, null, (ulong?)caseToUpdate.PriorityId, null, null, caseToUpdate.References, 
+                var updateCaseResult = policy.Execute(() =>
+                    _testRailClient.UpdateCase(caseId, caseToUpdate.Title, null, (ulong?)caseToUpdate.PriorityId, null, null, caseToUpdate.References,
```

Comment:
In `AddCase()` above you removed `PriorityId` cast to `ulong`, but in this method you still cast it. Is it intended?

Reply:
Fixed